twitter-lm.html

```diff
@@ -0,0 +1,571 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="twitter-lm.css" />
+    <script
+    src="https://kit.fontawesome.com/f9c0780752.js"
+    crossorigin="anonymous"></script>
+    <title>Twitter Clone</title>
+  </head>
+  <body>
+
+    <div class="container-wrapper">
+      <div class="container three-column-grid__expand-two center-wrapper-md">
+
+        <aside class="left-side">
+          <div class="logo">
+            <i class="fa-brands fa-twitter"></i>
```

Comment:
I've never seen i tags for icons, nice for structuring and separating icons from images. Will search them on MDN.

Reply:
Thank you, it helped readability for me and benefitted my structuring.